crates/apub/src/objects/community.rs

```diff
@@ -107,8 +108,10 @@ impl Object for ApubCommunity {
       id: self.id().into(),
       preferred_username: self.name.clone(),
       name: Some(self.title.clone()),
-      summary: self.description.as_ref().map(|b| markdown_to_html(b)),
-      source: self.description.clone().map(Source::new),
+      content: self.sidebar.as_ref().map(|d| markdown_to_html(d)),
+      source: self.sidebar.clone().map(Source::new),
+      summary: self.description.clone(),
+      media_type: self.sidebar.as_ref().map(|_| MediaTypeHtml::Html),
```

Comment:
This renaming will also cause some problems with backwards compat. But it does make more sense with the field definitions, so we probably need to do it this way (also for api).

Reply:
> Also they will try to update description with text that is too long. As the field name and type are unchanged, this will also be hard to notice during testing. So for backwards compat it would be better to leave description unchanged, and use a new name for the short description.

Couldn't we truncate to account for that? To me having these fields named different things for instance and community would be a bit scatterbrained, and if we don't make them consistent now, we never will.

Reply:
Truncating would make sense, see https://github.com/LemmyNet/lemmy/issues/1375

Reply:
Hrm... we should probably do these all at once then. So for all incoming federation receives, truncate on limited fields before inserting / updating?

Not sure if that should be done in this PR or a separate one.

Reply:
Exactly, and better in a separate PR.